### PR TITLE
Benchmark: Fixes configuration to ignore case

### PR DESCRIPTION
**Pull request by j82w**
# Pull Request Template

## Description

The input parameter logic will now ignore case. Before it was required to be all lower case. Fixes a bug to make the result parameters public as internal parameters are ignored.

## Type of change

Please delete options that are not relevant.

- [] Bug fix (non-breaking change which fixes an issue)
- [] New feature (non-breaking change which adds functionality)
- [] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [] This change requires a documentation update

## Closing issues

To automatically close an issue: closes #IssueNumber

### CODE REVIEW

File: Microsoft.Azure.Cosmos.Samples/Tools/Benchmark/Program.cs [hunk 1]
```diff
@@ -134,12 +141,11 @@ private async Task<RunSummary> ExecuteAsync(BenchmarkConfig config)
                 if (config.CleanupOnFinish)
                 {
                     Console.WriteLine($"Deleting Database {config.Database}");
-                    Microsoft.Azure.Cosmos.Database database = cosmosClient.GetDatabase(config.Database);
                     await database.DeleteStreamAsync();
                 }
 
                 runSummary.WorkloadType = config.WorkloadType;
-                runSummary.id = $"{DateTime.UtcNow.ToString("yyyy-MM-dd:HH-mm")}-{config.CommitId}";
+                runSummary.id = $"{DateTime.UtcNow:yyyy-MM-dd:HH-mm}-{config.CommitId}";
```
Comment (kirankumarkolli) [inline Microsoft.Azure.Cosmos.Samples/Tools/Benchmark/Program.cs:148]: It this syntax only applicable to ToString()

File: Microsoft.Azure.Cosmos.Samples/Tools/Benchmark/BenchmarkConfig.cs [hunk 2]
```diff
@@ -91,17 +91,17 @@ public class BenchmarkConfig
         public bool DisableCoreSdkLogging { get; set; }
 
         [Option(Required = false, HelpText = "Endpoint to publish results to")]
-        internal string ResultsEndpoint { get; set; }
+        public string ResultsEndpoint { get; set; }
```
Comment (kirankumarkolli) [inline Microsoft.Azure.Cosmos.Samples/Tools/Benchmark/BenchmarkConfig.cs:94]: Are these intentional?

**Review by kirankumarkolli — APPROVED** [review]

File: Microsoft.Azure.Cosmos.Samples/Tools/Benchmark/BenchmarkConfig.cs (hunk 2 above)
Comment (j82w) [reply Microsoft.Azure.Cosmos.Samples/Tools/Benchmark/BenchmarkConfig.cs:94]: Yes, it's described in the comments. This allows you set a different account/database/container to send the results to.

File: Microsoft.Azure.Cosmos.Samples/Tools/Benchmark/Program.cs (hunk 1 above)
Comment (j82w) [reply Microsoft.Azure.Cosmos.Samples/Tools/Benchmark/Program.cs:148]: I believe so, it was a VS suggestion.

Comment (j82w) [reply Microsoft.Azure.Cosmos.Samples/Tools/Benchmark/Program.cs:148]: https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/tokens/interpolated

**Pull request merged**